arangod/RocksDBEngine/RocksDBSettingsManager.cpp

```diff
@@ -208,7 +208,7 @@ Result RocksDBSettingsManager::sync(bool force) {
   }
 
   _tmpBuilder.clear();
-  Result res = writeSettings(batch, _tmpBuilder, minSeqNr);
```

Comment:
how is it possible that `_lastSync > minSeqNr` ?

Reply:
Previously, it was possible for `minSeqNr` to end up being 0. I observed it in practice. I believe it was the case where an index estimator was initialized with `_committedSeq` 0 and didn't have any work to do to serialize, so it just spit out the 0 (which then got passed down through all the `std::min`s).